src/extension/options-page/DashboardDialogs/DialogBase.tsx

```diff
@@ -136,8 +148,7 @@ export function DialogRouter(props: DialogRouterProps) {
             classes={{ paper: classes.dialog }}
             TransitionComponent={Transition}>
             <Route path={matchPattern?.path}>
-                {Component && <Component></Component>}
-                {children || null}
+                <ChildrenComponent />
```

Comment:
Is there any approach to ensure **ChildrenComponent** DO NOT re-render on `routeMatching` change?